src/Symfony/Component/Validator/Resources/translations/validators.es.xlf

```diff
@@ -325,6 +325,10 @@
             <trans-unit id="84">
                 <source>This value should be a multiple of {{ compared_value }}.</source>
                 <target>Este valor debería ser múltiplo de {{ compared_value }}.</target>
+            </trans-unit>            
+            <trans-unit id="85">
+                <source>This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.</source>
+                <target>Este Código de Identificación Bancaria (BIC) no está asociado con el IBAN {{iban}}</target>
```

Comment:
Please, maintain the white spaces around the `iban` placeholder name: `{{iban}}` -> `{{ iban }}`  Thanks!

Reply:
You're right @javiereguiluz , it's done.